packages/neuron-wallet/electron-builder.test.yml

```diff
@@ -0,0 +1,96 @@
+appId: com.nervos.neuron
+copyright: Copyright (C) 2019-2023 Nervos Foundation.
+productName: Neuron
+
+asar: false
+
+directories:
+  buildResources: .
+  output: ../../release
+
+afterSign: scripts/notarize.js
+
+files:
+  - from: "../.."
+    to: "."
+    filter: ["!**/*", ".ckb-version", ".ckb-light-version", "ormconfig.json", "compatible.csv"]
```

Comment:
The `compaitble.csv` should be `compatible.json`. And introducing one more template increases the burden of maintenance. How about setting some fields configurable and generating the configuration on the fly.

Reply:
Of course, if we import a lib that can parse and write a `yml` file to create the test yml from the original yml file, it will be better.